tests/tt_eager/integration_tests/test_bert.cpp

```diff
@@ -35,10 +35,12 @@ Tensor encoder(Tensor&& hidden_states, const Tensor& attention_mask, const Param
         .transpose_mcast = false,
         .fused_activation = std::nullopt,
     };
-    auto fused_qkv_matmul_output = tt::operations::primary::matmul(
+    auto fused_qkv_matmul_output = ttnn::operations::matmul::matmul(
         hidden_states,
         parameters.at(fmt::format("fused_qkv_weight_{}", encoder_index)),
         parameters.at(fmt::format("fused_qkv_bias_{}", encoder_index)),
+        /*transpose_a=*/false,
+        /*transpose_b=*/false,
```

Comment:
can we use enum flags instead?

So one can do 
`matmul(hiddent_states, params1, params2, Transpose::Both/None/A/B...)`
or
`matmul(hiddent_states, params1, params2, Transpose::A | Transpose::B)`

Reply:
(not a matter of this PR)

Reply:
Definitely not a matter for this PR.
The parameters were added as part of https://github.com/tenstorrent/tt-metal/pull/9836